explorer/src/components/account/BlockhashesCard.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import {
+  SysvarAccount,
+  RecentBlockhashesInfo,
+  RecentBlockhashesEntry,
+} from "validators/accounts/sysvar";
+
+export function BlockhashesCard({
+  sysvarAccount,
+}: {
+  sysvarAccount: SysvarAccount;
+}) {
+  const blockhashes = sysvarAccount.info as RecentBlockhashesInfo;
```

Comment:
How about passing `data.parsed.info` directly to this component so that you don't need a cast here?